quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -63,14 +62,10 @@ object VitaminQuantityPickers {
         ripple: RippleTheme = VitaminTheme.ripples.brand,
         onAddClicked: () -> Unit,
         onSubtractClicked: () -> Unit,
-        onValueChange: (String) -> Unit
+        onValueChange: (String) -> Unit,
     ) {
         CompositionLocalProvider(LocalRippleTheme provides ripple) {
-            Row(
-                modifier = modifier
-                    .height(48.dp)
-                    .width(154.dp)
-            ) {
+            Row(modifier = modifier.height(48.dp)) {
```

Comment:
We may need to add min width after design approval. PR is blocked until that

Reply:
done with : 
.widthIn(min = 154.dp)